test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNetCore/AspNetCoreWebApi31Tests.cs

```diff
@@ -0,0 +1,28 @@
+using Datadog.Trace.ClrProfiler.IntegrationTests.Helpers;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Datadog.Trace.ClrProfiler.IntegrationTests.AspNetCore
+{
+    public class AspNetCoreWebApi31Tests : AspNetCoreMvcTestBase
+    {
+        public AspNetCoreWebApi31Tests(ITestOutputHelper output)
+            : base("AspNetCoreWebApi31", output)
+        {
+            Expectations.Clear();
+            // TODO: When the automatic instrumentation begins to create traces from this sample,
+            // uncomment and remove the requestPathsOverride in MeetsAllAspNetCoreMvcExpectations
+            // CreateTopLevelExpectation(url: "/WeatherForecast", httpMethod: "GET", httpStatus: "200", resourceUrl: "WeatherForecast");
+        }
+
+        // TODO: Remove Skip property to run the sample in CI
+        [TargetFrameworkVersionsFact("netcoreapp3.1")]
+        [Trait("Category", "EndToEnd")]
+        [Trait("RunOnWindows", "True")]
+        public void MeetsAllAspNetCoreMvcExpectations()
+        {
+            // No package versions are relevant because this is built-in
+            RunTraceTestOnSelfHosted(string.Empty, requestPathsOverride: new[] { "/WeatherForecast" });
```

Comment:
This is a sort of weird workaround because the `AspNetCoreMvcBase` test class assumes a 1:1 ratio of web requests to generated spans. This allowed me to break that assumption so I could have one outbound request but above you can see I cleared out the list of Expectations so we can validate that no spans were created. This is the only place this new parameter is used. I'm happy to refactor this if there is a better way to achieve this.